Tasks/PhoneGapCommand/task.json

```diff
@@ -0,0 +1,72 @@
+{
+    "id": "61EC6339-4A6A-4A69-875E-5A6f4DB157A2",
+    "name": "PhoneGapCommand",
+    "friendlyName": "PhoneGap Command",
+    "description": "Run a PhoneGap CLI command",
+    "helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=692058)",
+    "category": "Build",
+    "visibility": [
+                "Build"
+                  ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 1
+    },
+    "demands": [
+        "npm"
+    ],
+    "instanceNameFormat": "PhoneGap Command $(phonegapCommand)",
+     "groups": [
+        {
+            "name":"advanced",
+            "displayName":"Advanced",
+            "isExpanded":false
+        }
+    ],
+    "inputs": [
+        { 
+            "name": "phonegapCommand", 
+            "type": "string", 
+            "label": "Command", 
+            "defaultValue":"", 
+            "required":true, 
+            "helpMarkDown": "PhoneGap command to run. Ex: \"platform add android\""
```

Comment:
Check the syntax here... it may need to be slightly different like "local platform add android"